doc/source/netscriptfunctions.rst

```diff
@@ -437,6 +437,30 @@ getHackingMultipliers
         print(mults.chance);
         print(mults.growth);
 
+getHacknetMultipliers
+^^^^^^^^^^^^^^^^^^^^^
+
+.. js:function:: getHacknetMultipliers()
+
+    Returns an object containing the Player's hacknet related multipliers. These multipliers are
+    returned in integer forms, not percentages (e.g. 1.5 instead of 150%). The object has the following structure::
```

Comment:
s/integer forms/fractional forms/

Reply:
tbh I had seen that but I copied from the other multipliers function as that's the typo there as well. I'll fix both I guess